src/components/editor-page/render-context/iframe-editor-to-renderer-communicator-context-provider.tsx

```diff
@@ -5,34 +5,29 @@
  */
 
 import React, { createContext, useContext, useMemo } from 'react'
-import { IframeEditorToRendererCommunicator } from '../../render-page/iframe-editor-to-renderer-communicator'
+import { EditorToRendererCommunicator } from '../../render-page/window-post-message-communicator/editor-to-renderer-communicator'
 
-const IFrameEditorToRendererCommunicatorContext = createContext<IframeEditorToRendererCommunicator | undefined>(
-  undefined
-)
+const IFrameEditorToRendererCommunicatorContext = createContext<EditorToRendererCommunicator | undefined>(undefined)
 
 /**
- * Provides the {@link IframeEditorToRendererCommunicator editor to renderer iframe communicator} that is set by a {@link IframeEditorToRendererCommunicatorContextProvider context provider}.
+ * Provides the {@link EditorToRendererCommunicator editor to renderer iframe communicator} that is set by a {@link EditorToRendererCommunicatorContextProvider context provider}.
  *
  * @return the received communicator
  * @throws Error if no communicator was received
  */
-export const useIFrameEditorToRendererCommunicator: () => IframeEditorToRendererCommunicator = () => {
+export const useEditorToRendererCommunicator: () => EditorToRendererCommunicator = () => {
   const communicatorFromContext = useContext(IFrameEditorToRendererCommunicatorContext)
   if (!communicatorFromContext) {
     throw new Error('No editor-to-renderer-iframe-communicator received. Did you forget to use the provider component?')
   }
   return communicatorFromContext
 }
 
-export const IframeEditorToRendererCommunicatorContextProvider: React.FC = ({ children }) => {
-  const currentIFrameCommunicator = useMemo<IframeEditorToRendererCommunicator>(
-    () => new IframeEditorToRendererCommunicator(),
-    []
-  )
+export const EditorToRendererCommunicatorContextProvider: React.FC = ({ children }) => {
```

Comment:
The exported component is called EditorToRendererCommunicatorContextProvider. The filename is different.